docs/tutorials/1.rst

```diff
@@ -15,9 +15,24 @@ In this first example, we will be calculating and printing the results of a Fibo
 specified number of elements. Python has some great standardized practices for iterating over large
 sequences that you will have an opportunity to learn about.
 
+This tutorial supports two modes of development: Github codespaces and python venv module.
+Feel free to choose the option more suitable to you below.
+
 Development Environment Setup
 -----------------------------
 
+Codespaces Environment
+^^^^^^^^^^^^^^^^^^^^^^
+
+To create a new codespace, click this button, and follow the prompts. It should take approximately 1-2 minutes to setup.
+
+.. raw:: html
+
+  <a href='https://codespaces.new/bloomberg/memray?devcontainer_path=.devcontainer%2Ftutorials%2Fdevcontainer.json'><img src='https://github.com/codespaces/badge.svg' alt='Open in GitHub Codespaces' style='max-width: 100%;'></a>
```

Comment:
This is how we have the github UI load with the correct codespace preselected. This won't work until merged in since it is pointing to the main repo/branch of Memray, but for a sample you can refer to my branch via this link:

https://codespaces.new/jcarnaxide/memray/tree/codespaces-for-tutorials?devcontainer_path=.devcontainer%2Ftutorials%2Fdevcontainer.json

Reply:
![Snag_30c72e](https://github.com/bloomberg/memray/assets/19931720/327f583e-440e-49de-9a5d-05b75ca58b3b)